presto-docs/src/main/sphinx/release/release-0.178.rst

```diff
@@ -0,0 +1,32 @@
+=============
+Release 0.178
+=============
+
+General Changes
+---------------
+
+* Fix various memory accounting bugs, which reduces the likelihood of full GCs/OOMs
+* Fix a regression that causes queries that use the keyword "stats"
+  to fail to parse.
+* Fix an issue where a query does not get cleaned up on the coordinator after query failure.
+* Add ability to cast from `REAL`/`TINYINT`/`SMALLINT` to JSON
+* Add support for :func:`GROUPING`.
```

Comment:
I'm not happy with this being in the aggregation functions documentation. It definitely shouldn't be shoved in the middle of  "General Aggregate Functions". It's not really an aggregation function. We're also missing a link to it from the SELECT documentation.

My thought is we move the documentation about it to be under "Complex Grouping Operations" and don't even document it as a function (since it's really not a function at all). That will make the documentation more coherent.